src/System.Diagnostics.PerformanceCounter/tests/PerformanceCounterCategoryTests.cs

```diff
@@ -1,17 +1,97 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
-
+//#define MyTrait
 using System;
 using System.Collections;
+using System.Collections.Generic;
 using System.Collections.Specialized;
+using System.Threading;
 using Xunit;
 
 namespace System.Diagnostics.Tests
 {
     [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // In appcontainer, cannot write to perf counters
     public static class PerformanceCounterCategoryTests
     {
+#if MyTrait
```

Comment:
Looks like MyTrait was left in inadvertently.

Reply:
As i have written above this is test/debug code, this test will be removed, or do you think we need an outerloop test of some seconds?

Reply:
I see.  I do like the idea of your multi-threaded test making making sure that we haven't introduced any deadlocks with this (or future) changes.  So, if you were to modify your test just to make sure that it actually completes within a reasonable time period, I think that would be a good addition to the test suite.

Reply:
ok if the idea of some sort of "wait" will be accepted i refactor an [OuterLoop()] test.

Reply:
Makes sense.